pdns/lua-recursor4.cc

```diff
@@ -460,6 +460,14 @@ void RecursorLua4::postLoad() {
   d_gettag_ffi = d_lw->readVariable<boost::optional<gettag_ffi_t>>("gettag_ffi").get_value_or(0);
 }
 
+void RecursorLua4::getFeatures(Features & features) {
+  // Add key-values pairs below.
+  // Make sure you add string values explicity converted to string.
+  // e.g. features.push_back(make_pair("somekey", string("stringvalue"));
+  // Both int and double end up as a lua number type.
+   features.push_back(make_pair("PR8001_devicename", true));
```

Comment:
Just a nit but I would have expected this function to return a `Features` object, perhaps as a const reference, rather than take it as an out parameter. That doesn't matter, though.

Reply:
I have done it this way to allow for a more easy way to add features in several places. Not needed now, but maybe in the future. Each method call will then add its features to an existing vector.